server/api/index.js

```diff
@@ -1,3 +1,24 @@
-const router = require('express').Router(); 
+const router = require('express').Router();
+const rp = require('request-promise');
+const Nationality = require ('../../db/model/Nationality.js')
+const { formatCountryText }  = require('../../utility/util')
+
+
+
+router.post('/', (req, res, next) => {
```

Comment:
I can deal with this in the future but instead of having a router dealing with a req.body, we should honestly just have back end util functions that get run. There shouldn't be a route on our server triggering this to occur.